Dockerfile

```diff
@@ -8,4 +8,4 @@ RUN cd /src && ./build.sh "$(cat VERSION)"
 
 ONBUILD COPY ./build.sh /src/build.sh
 ONBUILD COPY ./modules.go /src/modules.go
-ONBUILD RUN cd /src && chmod +x ./build.sh && ./build.sh "$(cat VERSION)-custom"
+ONBUILD RUN cd /src && chmod +x ./build.sh ; sleep 1 && sync && ./build.sh "$(cat VERSION)-custom"
```

Comment:
does this work without the sleep?

Reply:
I don't think it does (consistently); [this build](https://hub.docker.com/r/billimek/logspout/builds/bhcbrrbwoeijrnbhhgznj5z/) failed but [this one](https://hub.docker.com/r/billimek/logspout/builds/bg7ymtnwbfamszz7cxsu7hp/) succeeded.

While the sleep + sync [succeeded](https://hub.docker.com/r/billimek/logspout/builds/brwm76fgojhxgbq6knyeuc4/).

I figured it is safer to do the `sleep 1` + sync than not.

Reply:
Any reason to not chain all of these commands with `&&`?

Reply:
No reason, and pushed a commit to make all commands chained with `&&`